client/state/plugins/installed/selectors-ts.ts

```diff
@@ -280,7 +282,7 @@ export const getPluginOnSite = createSelector(
 		};
 	},
 	( state: AppState ) => [ getAllPluginsIndexedByPluginSlug( state ) ]
-);
+) as ( state: AppState, siteId: number, pluginSlug: string ) => Plugin & PluginSite;
```

Comment:
I left this return type as `Plugin & PluginSite` instead of declaring that a different type as I don't think this pattern is ideal, so I think it's best if this type isn't named. It should be clear that these functions are returning a combination of two types.